docs/questions/query-builder/expressions/converttimezone.md

```diff
@@ -0,0 +1,139 @@
+- **Metabase report time zone**: the time zone that Metabase uses to _display_ datetime data. For example, if your database time zone is in UTC, your Metabase admin can choose to display all dates and times in PST.
+
+You should only change these source time zones to target time zones if it'll help people interpret the data:
+
+- A custom report time zone (for example, say you live in HKT, but you're making a report for a team in EST).
+- Local time, if it's important for people to see data in the time zone that they live in.
+
+| Original Time                | Database Time                | Metabase Report Time         | Team Report Time (EST)       |  Local Report Time (GMT)     |
+|------------------------------|------------------------------|------------------------------|------------------------------|------------------------------|
+| December 28, 2022, 00:00:00  | December 28, 2022, 18:00:00  | December 28, 2022, 10:00:00  | December 28, 2022, 07:00:00  | December 28, 2022, 12:00:00  |
+| December 28, 2022, 00:00:00  | December 28, 2022, 05:00:00  | December 28, 2022, 21:00:00  | December 28, 2022, 19:00:00  | December 29, 2022, 00:00:00  |
+| December 28, 2022, 00:00:00  | December 27, 2022, 16:00:00  | December 27, 2022, 08:00:00  | December 27, 2022, 05:00:00  | December 27, 2022, 10:00:00  |
+
+For example, you could create **Team Report Time (EST)** differently depending on the source:
+
+1. **Original Time**
+
+```
+convertTimezone([Original Time)], 'EST')
+```
+
+2. **Database Time** (say you know your database is in UTC)
+
+```
+convertTimezone([Database time (UTC)], 'EST', 'UTC')
+```
+
+3. **Metabase Report Time** (if you know that Metabase reports in PST)
+
+```
+convertTimezone([Metabase Report Time (PST)], 'EST', 'PST')
+```
+
+Once you've created **Team Report Time (EST)**, you could convert it again to a **Local Report Time (GMT)** (maybe for a teammate on the move):
+
+```
+convertTimezone([Team Report Time (EST)], 'GMT', 'EST')
+```
+
+Before you do time zone conversions, make sure you know the source time zone that you're working with. The example here shows you how the source time zones can be different depending on the system that it comes from (original event capture, database, Metabase), but your datasets might not make it so obvious. 
+
+When you make a custom column with a new time zone, it's usually a good idea to name the column with that time zone. We promise that this a life a lot easier when someone inevitably asks why certain numbers don't match.
+
+## Accepted data types
+
+| [Data type](https://www.metabase.com/learn/databases/data-types-overview#examples-of-data-types) | Works with `convertTimezone`  |
+| ----------------------- | -------------------- |
+| String                  | ❌                   |
+| Number                  | ❌                   |
+| Timestamp               | ✅                   |
```

Comment:
can we make a distinciton between `timestamp with time zone` and `timestamp without time zone`?

Reply:
This table is the same across all the custom expression docs. It's just there to help people do a quick check for common data type errors, so won't expand it into more detail than this (otherwise we could make a case to split out the other data types too!).

I'll add a note that the function will error if you provide a `source` parameter on `timestamp with time zone`.